quad_control/scripts/TrajectoryPlanner/trajectory.py

```diff
@@ -4,39 +4,36 @@
 
 
 import numpy as np
-import utilities.utility_functions as uts
+from utilities import utility_functions as uts
 
 
 class Trajectory:
 
 
-    PARAMETERS_NAMES = ()
```

Comment:
Variables in this position are static, which means that they cannot be overrided by child classes.
This caused me some trouble when initializing objects with dictionaries, since I was using these variables to hold the strings corresponding to the parameters names (for example `PARAMETERS_NAMES=('radius','speed')`.
This led me to take away the option of initializing objects with dictionaries.
Instead, I went back to the initialization with ordered lists.

This long comment was done mostly because I found that we can comment single lines of code, and I was to happy not to use the feature right away u.u